lessons/Data-Science/AI/ai-training/langs/en-us/pages/human-judgment.adoc

```diff
@@ -0,0 +1,69 @@
+= The Importance of Human Judgment
+
+Open the @starter-file{plagiarism}, click "Run", and then respond to the questions below.
+
+== Determine Closeness
+
+@n Use `cosine-similarity` to compare the student-submitted essay with the wikipedia article. Click "Run". What does the program tell you about the comparison? @fitb{}{@ifsoln{`cosine-similarity(student-essay, wiki-article)` produces an output of ~0.97087...,}}
```

Comment:
before students run the expression in the file, we should definitely ask them what they would want a good plagiarism detector to return!